package.json

```diff
@@ -40,6 +40,7 @@
   "resolutions": {
     "babel-core": "^7.0.0-0",
     "graphql": "^0.13.2",
+    "moment": "^2.22.2",
```

Comment:
So different packages were depending on different versions that could not be resolved in a single dependency graph?

Reply:
Yep